gpu/containers/include/pcl/gpu/containers/device_memory.h

```diff
@@ -95,12 +95,36 @@ namespace pcl
               * \param sizeBytes_arg buffer size
               * */
             void upload(const void *host_ptr_arg, std::size_t sizeBytes_arg);
-            
+
+            /** \brief Uploads data from CPU memory to device array. Please note
+             * that this overload never allocates memory in contrast to the
+             * other upload function.
+             * Returns true if upload successfull
```

Comment:
```suggestion
             * Returns true if upload successful
```
Once more below, also consider doxygen's `\return` tag

Reply:
Thanks!

Reply:
This should be fixed - thanks for the comment!